tidy up (multiple changes)

leokalev opened the pull request:
(note other changes are desirable, e.g., better error handling, as noted in comments in the code. This branch could be updated further before we merge it)

- cmd line help messages
- rename 'measure_*' functions, out of place in 'adjust'
- send adjust data as arg to self.adjust()

Commit:
tidy up (multiple changes)

- cmd line help messages
- rename 'measure_*' functions, out of place in 'adjust'
- send adjust data as arg to self.adjust()

pavel-georgiev commented on adjust.py:
```diff
@@ -81,10 +87,11 @@ def run(self):
-            self.input_data = json.loads(sys.stdin.read())
+            # self.debug("Reading stdin")
+            input_data = json.loads(sys.stdin.read())
+            self.input_data = input_data # LEGACY mode, remove when drivers are updated to use arg
```
Shouldn't this also come with a change in the prototype for the base adjust() method, i.e. `adjust(input=None)`?

Commit:
doc: adjust() method signature: add 'data'

this is for documentation purposes only, no functional change,
support for adjust with either the old or the new signature is
retained.
